data/metadata.json

```diff
@@ -121,6 +123,11 @@
     "codeview/popup/fix": {
         "sound-file": "codeview/errorPopup - fixitButtonPress.webm"
     },
+    "fizzics/background/back": {
```

Comment:
This can be removed, it's already in https://github.com/endlessm/hack-sound-server/pull/91